src/Microsoft.Data.SqlClient/tests/FunctionalTests/AlwaysEncryptedTests/Utility.cs

```diff
@@ -348,8 +348,9 @@ internal static byte[] DecryptDataUsingAED(byte[] encryptedCellBlob, byte[] key,
             return decryptedData;
         }
 
+#if NETFRAMEWORK
         /// <summary>
```

Comment:
This method is only called in netfx. In net5, accessing the PrivateKey property throws an exception. Now in net6, it won't even compile, so we can just compile it for netfx.